ligo/src/common.mligo

```diff
@@ -0,0 +1,19 @@
+// SPDX-FileCopyrightText: 2020 TQ Tezos
+// SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+#if !COMMON_H
+#define COMMON_H
+
+#include "types.mligo"
+
+let authorize_admin (store: storage): bool =
+    store.admin = Tezos.sender
```

Comment:
While this should be here (as it will be used by `token` entrypoints too) I suggest we modify it to match the version one @rinn7e made for it there, in #104 :
```suggestion
let authorize_admin (store : storage): storage =
  if sender = store.admin then
    store
  else
    (failwith("NOT_ADMIN"): storage)
```
because:
1. we are trying to make this check on `storage` data all have it returned (see [discussion here]())
2. it doesn't make sense to only make the boolean check and write the `failwith` elsewhere, because the latter is always going to be the same.

Also, we should check if this should be `[@inline]`d as well, it would surely make sense in it's current implementation, but probably still will in the suggested one.

P.s. @rinn7e since I have already tagged you, please review this PR as well when you can.